src/main/java/ex/google/faculty_schedule_preference/user_availability/UserAvailabilityController.java

```diff
@@ -79,11 +94,22 @@ public String myAvailabilityNew(Model model, @AuthenticationPrincipal UserDetail
       // Post: api/UserAvailability/Create
       @PostMapping("my_availability/create")
       public String myAvailabilityCreate(@ModelAttribute UserAvailability userAvailability,
-                  @AuthenticationPrincipal UserDetails userDetails,
+                  @AuthenticationPrincipal UserDetails userDetails, @RequestParam("releaseTimeArray") String [] releaseTimeArray,
                   RedirectAttributes redirectAttributes) {
             User currentUser = userRepository.findByUsername(userDetails.getUsername()).get();
+            
             userAvailability.setUser(currentUser);
             repository.save(userAvailability);
+
+            for (int i = 0; i < releaseTimeArray.length; i +=3){
```

Comment:
why i+=3
does the array store empty elements.

Reply:
i +=3 is because all the data relevant to release times is stored in array (units, source and note)